config/manager/default-monitoring.yaml

```
@@ -83,6 +83,7 @@ data:
           , pg_catalog.age(datfrozenxid) AS xid_age
           , pg_catalog.mxid_age(datminmxid) AS mxid_age
         FROM pg_catalog.pg_database
+          WHERE datallowconn = 't'
```

Nit picking (indentation and a boolean doesn't need to be compared to true)
```suggestion
        WHERE datallowconn
```